ntbs-service/Pages/_HomepageKpiDetails.cshtml

```diff
@@ -0,0 +1,48 @@
+﻿@model ntbs_service.Pages.IndexModel
+
+<div>
+    <filtered-homepage-kpi inline-template>
+        <div id="homepage-kpi-details">
+            <div class="flex-container">
+                <label nhs-label-type="Standard" for="code-filter-dropdown" classes="homepage-kpi-filter-label" >
+                    Filter by code
+                </label>
+                <select id="code-filter-dropdown" nhs-select-type="Standard" ref="filterByCode" asp-items="Model.Codes"
+                    v-on:change="updateKpiDetails" classes="homepage-kpi-filter-dropdown">
+                </select>
+            </div>
+            <br/>
+            @foreach (var item in Model.HomepageKpiDetails)
+            {
+                <div id="kpi-@item.Code" v-if="CodeFilter == '@item.Code'">

```

Comment:
Is the intended non-JS behaviour for this to just render all the KPIs on top of each other?

Reply:
I have taken into account non js users as well